### DISCUSSION TITLE
Pull comment sort menu out of dropdown

### DISCUSSION THREAD

**Discussion by omentic** [Docile-Alligator/Infinity-For-Reddit#930] [Ideas]
When on a post screen, it'd be cool if the comment sort option was pulled out of the dropdown menu into its own button. I've attached before / after mockups below:

![Screenshot from 2021-03-18 17-51-35](https://user-images.githubusercontent.com/35242550/111716155-aab96e00-8812-11eb-919c-05ba0b811331.png)
![Screenshot from 2021-03-18 17-51-49](https://user-images.githubusercontent.com/35242550/111716152-aa20d780-8812-11eb-8c64-97d95fe4aa6c.png)

Sorting comment threads is a much more common thing to do than any of the other options in the dropdown menu - and adding a sort button would also make it fit thematically with the subreddit bar.

**Comment by Docile-Alligator** [comment]
Thank you for this idea!
